.github/workflows/test.yml

```diff
@@ -50,8 +50,8 @@ jobs:
       - run: npm install --ignore-scripts
       - run: npm run test:cli --workspace="@csstools/postcss-base-plugin"
 
-  # Run Deno tests one by one
-  test_deno:
+  # Run e2e tests
```

Comment:
Deno tests are also a form of e2e.
So made sense to me to group these.

One less full build of all plugins if we have then in a single job.